src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -348,6 +341,19 @@ private List<CastVoteRecord> parseCastVoteRecords(ContestConfig config) {
       }
     }
 
+    // Output the RCTab-CSV CVR
```

Comment:
Is this just code reorganization or is there a more substantive reason that you moved this down a bit? A little hard to tell just from viewing this here.

Reply:
Before it was inside the for-loop (i.e. per-source); now it's once per tabulator session